src/main/resources/META-INF/plugin.xml

```diff
@@ -1,13 +1,14 @@
 <idea-plugin>
-    <id>io.coala.jetbrains.code-analysis</id>
+    <id>io.coala.jetbrains</id>
     <name>coala Code Analysis</name>
     <vendor url="https://coala.io/">coala</vendor>
 
     <description><![CDATA[
-    <h1>coala: Language Independent Code Analysis</h1>
     coala provides a unified interface for linting and fixing all your code, regardless of the programming languages you use.
     ]]></description>
 
+    <change-notes><!-- --></change-notes>
```

Comment:
`<!-- -->` needed?

Reply:
the `pathPluginXml` from `build.gradle` will over-write this